src/app/search-page/filter-component/filter.component.scss

```diff
@@ -0,0 +1,33 @@
+$mat-select-bg: rgb(255, 255, 255);
+$wrapper-bg: rgb(241, 239, 239);
+$box-shadow: 0 0 1.25rem #0000001a;
+
+#container-select {
+  background-color: $mat-select-bg;
+  box-shadow: $box-shadow;
+}
+.sticky {
+  position: sticky;
+  position: -webkit-sticky;
+  top: 64px;
+  z-index: 1000;
+}
```

Comment:
This class is also created in:
 header.component.scss
If you need the same css class in more then one component it makes sense to place it in a "global" css file. It should in this path: "src/styles.scss"